conference.js

```diff
@@ -1548,7 +1550,7 @@ export default {
             // again switching to the screen sharing.
             APP.UI.showExtensionInlineInstallationDialog(
                 () => {
-                    this.toggleScreenSharing();
+                    this.toggleScreenSharing().catch(() => {});
```

Comment:
You should be able to test that by adding extension id from meet.jit.si to you dev config. I think it should still work, but would be better to try it just to be sure.